espnet/nets/pytorch_backend/rnn/decoders_transducer.py

```diff
@@ -356,7 +356,7 @@ def zero_state(self, ey):
         """Initialize decoder states.
 
         Args:
-            ey (torch.Tensor): batch of input features (B, Lmax, Emb_dim)
+            ey (torch.Tensor): batch of input features (B, Lmax, X_dim)
```

Comment:
I modified this docstring because in some cases I'm using `hs_pad`, which can have a different dimension, for initialization. It is because I'm modyfying `ey` type and a `type error` is thrown in some calculations. As seen in the code, the third dimension is not needed for initialization.
Is that all right?